index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap"
+    rel="stylesheet" />
+</head>
+<body>
+   
+    <header>
+        <!-- <div class="topbar"><img src="images/topbar.png" alt=""></div> -->
+        <div class="logo">
+            <p>Hasenat </p></div>
+            <div class="icon">
+            <a href="#"><img src="images/Icon - Menu.png" alt=""></a>
+        </div>
+        
+    </header>
+    <section class="headline">
+        <h1 class="name">Hey There.<br /> <span>I'm Hasenat</span></h1>
+        <h3>I'm a Software Developer</h3>
```

Comment:
- [ ] I would like you to consider using `Roboto` font-family for this subtitle on your page as to be inline with what we have on the guiding template. Kindly visit `Google Fronts` for that font-family and use it for this element.